app-analyses/src/main/kotlin/com/usvision/analyses/analyzer/NumberOfDependents.kt

```diff
@@ -0,0 +1,33 @@
+package com.usvision.analyses.analyzer
+
+import com.usvision.model.domain.Microservice
+import com.usvision.model.visitor.Visitable
+
+class NumberOfDependents(
+    private val asyncDependenciesOfMicroservice: AsyncDependenciesOfMicroservice,
+    private val syncDependenciesOfMicroservice: SyncDependenciesOfMicroservice
+) : Measurer() {
+    private val INT_TYPE_NAME = Int::class.qualifiedName!!
+    private val UNIT_DEPENDENTS = "dependents"
+
+    private val counters: MutableMap<Visitable, Measure> = mutableMapOf()
+
+    override fun getResults(): Map<Visitable, Measure> = counters
+
+    override fun visit(microservice: Microservice) {
+        val asyncDependents = asyncDependenciesOfMicroservice.getResults().filter { (ms, _) ->
```

Comment:
Eu deixaria essa chamada do `.getResults()` direto no construtor e armazenaria o resultado em um atributo privado, assim evita fazer a chamada toda vez que um microservice for visitado

Reply:
Alterei para armazenar em variáveis, o que melhorou o número de chamadas.
Quando alterei para ficar direto no constructor, tive alguns problemas com a detecção.
Como já houve um ganho interessante somente com o armazenamento, mantive dessa maneira.